hbase-server/pom.xml

```diff
@@ -723,9 +723,6 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-distcp</artifactId>
-          <!--This dependency is not in top-level pom so needs
-               version specified-->
```

Comment:
Oh, one more thing. Please apply these same changes (picking up the dependency version from the parent pom) to the Hadoop2 profile.

Reply:
Hi Nick, not sure i get that. Both hadoop-distcp and hadoop-hdfs-client versions are specified in the hadoop-2 and hadoop-3 profiles in the root pom file.

Reply:
Looking at the `hadoop-2.0` profile in `hbase-server/pom.xml`, I see there's a version specified there for the `hadoop-distcp` dependency. You can apply the same change to this profile that you did to the `hadoop-3.0` profile, so that it will resolve the version from the parent pom's profile instead of the hard-coded version.

https://github.com/ndimiduk/hbase/blob/00fc46756abb99de6f833997499505f89c9752e8/hbase-server/pom.xml#L612-L618

Reply:
Ah. good catch. Updated the PR to add this. Thanks!